packages/react-dom/src/__tests__/ReactDOMComponent-test.js

```diff
@@ -1048,9 +1048,15 @@ describe('ReactDOMComponent', () => {
     }
 
     it('should generate the correct markup with className', () => {
-      expect(toHaveAttribute(genMarkup({className: 'a'}), ['class', 'a']));
-      expect(toHaveAttribute(genMarkup({className: 'a b'}), ['class', 'a b']));
-      expect(toHaveAttribute(genMarkup({className: ''}), ['class', '']));
+      expect(toHaveAttribute(genMarkup({className: 'a'}), ['class', 'a'])).toBe(
```

Comment:
not really related to this PR, but `toHaveAttribute` should probably be written as a custom matcher rather than a function which returns a boolean.